extras/systemd/dynflow-sidekiq@.service

```diff
@@ -4,13 +4,23 @@ Documentation=https://theforeman.org
 After=network.target remote-fs.target nss-lookup.target
 
 [Service]
-Type=forking
+Type=simple
 User=foreman
 TimeoutSec=90
 Environment=RAILS_ENV=production
 Environment=DYNFLOW_SIDEKIQ_SCRIPT=/usr/share/foreman/extras/dynflow-sidekiq.rb
+# Greatly reduce Ruby memory fragmentation and heap usage
+# https://www.mikeperham.com/2018/04/25/taming-rails-memory-bloat/
+Environment=MALLOC_ARENA_MAX=2
 WorkingDirectory=/usr/share/foreman
 ExecStart=/usr/bin/sidekiq -e ${RAILS_ENV} -r ${DYNFLOW_SIDEKIQ_SCRIPT} -C /etc/foreman/dynflow/%i.yml
```

Comment:
So one downside to this pattern is that if I give it a name that does not exist, it silently accepts my systemctl command leaving the user with no indication they did something wrong. Here is my example:

I accidentally misspell orchestrator trying to start or restart it:
```
[root@centos7-katello-nightly vagrant]# systemctl start dynflow-sidekiq@orchestrat
[root@centos7-katello-nightly vagrant]#
```

The output is silent leaving me with no clear indicator I tried to start something that does not exist. The journal does log it:

```
Oct 24 16:27:54 centos7-katello-nightly.war.example.com dynflow-sidekiq@orchestrat[28392]: No such file /etc/foreman/dynflow/orchestrat.yml
Oct 24 16:27:54 centos7-katello-nightly.war.example.com dynflow-sidekiq@orchestrat[28392]: /opt/theforeman/tfm/root/usr/share/gems/gems/sidekiq-5.2.7/lib/sidekiq/cli.rb:239:in `setup_options'
Oct 24 16:27:54 centos7-katello-nightly.war.example.com dynflow-sidekiq@orchestrat[28392]: /opt/theforeman/tfm/root/usr/share/gems/gems/sidekiq-5.2.7/lib/sidekiq/cli.rb:31:in `parse'
Oct 24 16:27:54 centos7-katello-nightly.war.example.com dynflow-sidekiq@orchestrat[28392]: /opt/theforeman/tfm/root/usr/share/gems/gems/sidekiq-5.2.7/bin/sidekiq:11:in `<top (required)>'
Oct 24 16:27:54 centos7-katello-nightly.war.example.com dynflow-sidekiq@orchestrat[28392]: /opt/theforeman/tfm/root/usr/bin/sidekiq:23:in `load'
Oct 24 16:27:54 centos7-katello-nightly.war.example.com dynflow-sidekiq@orchestrat[28392]: /opt/theforeman/tfm/root/usr/bin/sidekiq:23:in `<main>'
```

However, I am more than certain we will get bugs logged with this pattern. I think we should re-think this pattern or add some sort of checking that produces clear output to a user when managing these services that they tried to perform an operation on an unknown configuration.

Reply:
Noted, don't know what to do about it, though :thinking: I will try to figure something out. But would like to get this in 1.24 if possible and I believe this is improvement even without it.